packages/event-handler/src/rest/middleware/compress.ts

```diff
@@ -0,0 +1,58 @@
+import type { Middleware } from '@aws-lambda-powertools/event-handler/types';
```

Comment:
This should be a local relative import.

The rule of thumb is:
- if it's under the same `packages/<name>/src` folder, it uses a relative import 
- if it's under a different one, it uses a full named import (like what you put above)